Makefile

```
@@ -0,0 +1,6 @@
+TEST_DIR := ./agent/functions
+
+.PHONY: test
+
+test:
+	cd $(TEST_DIR) && go test -v
```

from CodeQualityAnalyst
Makefileのtestターゲットは、テストディレクトリに移動してからgo testを実行していますが、テストの実行結果をより詳細に把握するために、カバレッジレポートを生成するオプションを追加することをお勧めします。

```suggestion
test:
	cd $(TEST_DIR) && go test -v -cover
```